static/skywire-manager-src/src/assets/i18n/de.json

```diff
@@ -303,7 +430,13 @@
 
   "transports": {
     "title": "Transporte",
+    "remove-all-offline": "Alle offline Transporte löschen",
+    "remove-all-offline-confirmation": "Wirkliche alle offline Transporte löschen?",
+    "remove-all-filtered-offline-confirmation": "Alle offline Transporte, welche die Filterkriterien erfüllen werden gelöscht. Wirklich fortfahren?",
+    "info": "Verbindungen mit Remote Skywire Visor, um lokalen Skywire Anwendungen zu erlauben mit diesen remote Visor zu kommunizieren.",
```

Comment:
Capitaliazation of `remote` is not consistent.

Reply:
Changed all capitalized (Remote) to remote. Unless at the beginning of a sentence